metadata-ingestion/src/datahub/ingestion/run/pipeline.py

```diff
@@ -379,13 +379,19 @@ def _notify_reporters_on_ingestion_completion(self) -> None:
         for reporter in self.reporters:
             try:
                 reporter.on_completion(
-                    status="CANCELLED"
-                    if self.final_status == PipelineStatus.CANCELLED
-                    else "FAILURE"
-                    if self.has_failures()
-                    else "SUCCESS"
-                    if self.final_status == PipelineStatus.COMPLETED
-                    else "UNKNOWN",
+                    status=(
```

Comment:
Even though it is not too complex but looks complex this way

Reply:
this was just the autoformatting of my ide - but yeah we can tweak this in a follow up